gensim/models/ldamodel.py

```diff
@@ -820,6 +856,7 @@ def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True):
 
             # add a little random jitter, to randomize results around the same alpha
             sort_alpha = self.alpha + 0.0001 * self.random_state.rand(len(self.alpha))
+            # random_state.rand returns float64, but converting back to dtype won't speed up anything
```

Comment:
Maybe `.astype` (for consistency only) ?

Reply:
Consistency vs one additional array copy. I'm not sure :)